specification/amqp_0.9.1/README.md

```diff
@@ -0,0 +1,7 @@
+## AMQP 0.9.1 Protocol Specification
+
+This project uses [K3PO](http://github.com/k3po/k3po) to help AMQP 0.9.1 Protocol implementations automate their verification of the necessary specification requirements.
```

Comment:
Add a sentence saying something like "NOTE: only opening and closing handshake is covered for now"